.gitignore

```diff
@@ -2,14 +2,17 @@
 __pycache__
 /load_dataset_paths.json
 *.csv
+*.prof
 *.json*
 */tmp_*
-/datasets/TLSE_2022001/BRDC00IGS_R_20220010000_01D_MN.rnx
-/datasets/TLSE_2022001/BRDC00IGS_R_20220010000_01D_GN.rnx
-/datasets/TLSE_2022001/BRD400IGS_S_20220010000_01D_MN.rnx
-/datasets/TLSE_2022001/TLSE00FRA_R_20220010000_01D_30S_MO.rnx
-/datasets/TLSE_2022001/TLSE00FRA_R_20220010000_01D_30S_GO.rnx
+/src/prx/test/datasets/TLSE_2022001/BRDC00IGS_R_20220010000_01D_MN.rnx
```

Comment:
can we use a regex here, something like `/src/prx/test/datasets/*.rnx?` I think we ususally track only `.CRX.gz` files, right?

Reply:
I'll ignore all `*.rnx` and `*.crx` files.